### PR TITLE
Tiers get sent to Ostriz

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 43293d2d30e37d75f76d47b39364091fd4287c8e:

Everything seems all right :smile: .
*CFME QE Bot*

File: artifactor/plugins/reporter.py [hunk 1]
```diff
@@ -363,9 +363,11 @@ def skip_test(self, test_location, test_name, skip_data):
         return None, {'artifacts': {test_ident: {'skipped': skip_data}}}
 
     @ArtifactorBasePlugin.check_configured
-    def start_test(self, test_location, test_name, slaveid):
+    def start_test(self, test_location, test_name, slaveid, tier=None):
         test_ident = "{}/{}".format(test_location, test_name)
-        return None, {'artifacts': {test_ident: {'start_time': time.time(), 'slaveid': slaveid}}}
+        return None, {'artifacts': {test_ident: {
+            'start_time': time.time(), 'slaveid': slaveid, 'tier': tier or "N/A"}}
```
Comment (jkrocil) [inline artifactor/plugins/reporter.py]: I assume it's ok that `tier` is a number when set but a string (`'N/A'`) when not set?

Comment (psav) [inline artifactor/plugins/reporter.py]: Yeh that's fine

Comment (jkrocil) [inline artifactor/plugins/reporter.py]: Merging then

**Pull request merged**